mozdownload/treeherder.py

```diff
@@ -2,16 +2,19 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+"""Module to retrieve builds via revision."""
```

Comment:
Ouch! I accidentally landed this file already with https://github.com/mozilla/mozdownload/commit/fc8f22f6b26ed339c375f96fb9f008719f5b3cfa. Not sure why. :(